src/executables/util_cs.c

```diff
@@ -4000,6 +4000,58 @@ tde (UTIL_FUNCTION_ARG * arg)
   return EXIT_FAILURE;
 }
 
+static int
+check_table_name (const char *table_name)
+{
+  int table_name_len = STATIC_CAST (int, strlen (table_name));
+  int sub_len = 0;
+  const char *dot = NULL;
+
+  if (table_name_len >= SM_MAX_IDENTIFIER_LENGTH)
+    {
+      PRINT_AND_LOG_ERR_MSG (msgcat_message
+			     (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_FLASHBACK,
+			      FLASHBACK_MSG_EXCEED_MAX_OWNER_CLASS_LENGTH), SM_MAX_IDENTIFIER_LENGTH);
+
+      return ER_FAILED;
+    }
+
+  dot = strchr (table_name, '.');
+  if (dot == NULL)
+    {
+      /* owner name is not specified */
+      PRINT_AND_LOG_ERR_MSG (msgcat_message
+			     (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_FLASHBACK, FLASHBACK_MSG_OWNER_NOT_SPECIFIED),
+			     table_name);
+
+      return ER_FAILED;
+    }
+
+  /* check length of owner name */
+  sub_len = STATIC_CAST (int, dot - table_name);
+  if (sub_len < 1 || sub_len >= SM_MAX_USER_LENGTH)
```

Comment:
because, users can input class_name without owner name, and just add 'dot' to the class_name like below
`cubrid flashback demodb .test_table` 

So, I added a condition `sub_len<1`

Reply:
If then, you need to use 'sub_len <= 1'. isn't it?

Reply:
if owner name is 'a' , `sub_len` will be 1, and user 'a' could exist. 
`sub_len` means the length of owner_name.
length for a dot is not included in `sub_len`.

Reply:
Oh. I understand.